db/migrate/20161017135917_create_organizations.rb

```diff
@@ -0,0 +1,16 @@
+class CreateOrganizations < ActiveRecord::Migration
+  def change
+    create_table :organizations do |t|
+      t.string :name
+      t.string :display_name
+      t.string :slug, default: ""
+      t.string :primary_language, null: false
+      t.timestamp :listed_at, default: nil
+      t.integer  :activated_state, default: 0, null: false
+
+      t.timestamps null: false
+    end
+
+    add_index :organizations, :listed_at, where: "(listed_at IS NOT NULL)"
```

Comment:
not sure this partial index will satisfy both scope defs and thus be used for both, you might want to test it out using a normal btree index (no partial) in your test env without sequential scans disabled.